apps/exchange/lib/exchange.ex

```diff
@@ -93,6 +93,10 @@ defmodule Exchange do
     Exchange.MatchingEngine.ask_volume(ticker)
   end
 
+  def open_orders(ticker) do
+    Exchange.MatchingEngine.open_orders(ticker)
+  end
+
```

Comment:
Here you should have a call for the open_orders_trader_id.
The arguments are the ticker and trader_id